scripts/skills/actives/rf_inspire_skill.nut

```diff
@@ -0,0 +1,99 @@
+this.rf_inspire_skill <- this.inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		InspireBonusFraction = 0.1,		// This percentage of Resolve is added as a bonus for the morale check
+
+		// Private
+		InspireFlag = "RF_InspireSkillFlag",
+	},
+
+	function create()
+	{
+		this.m.ID = "actives.rf_inspire";
+		this.m.Name = "Inspire";
+		this.m.Description = "Inspire an ally to raise their current Morale. A character can only be inspired once per turn. Cannot be used on fleeing or stunned allies.";
```

Comment:
- `Morale` and `fleeing` should be hyperlinked with `Concept.Morale`.
- `turn` should be hyperlinked with `Concept.Turn`.
- `stunned` should be hyperlinked with stunned effect.